gravitee-io/gravitee-access-management
# AM-87 Update the certificate used by applications after a default cer…

…t renewal

# How to test

* Create a new domain
* create an app
* check the app certificate is set to Default
* go to domain settings > certificate
* click on "Rotate System Key"
* wait 10 minutes (default value for the application update delay)
* check the app certificate is set to "Default yyyy-MM-dd HH:mm:ss" 
* check into the audit logs of the domain that application has been updated by the actor "system" and into the audit detail, only the certificate field is present
* try to initiate the login flow for this application to get an access_token. The access-token should be signed using the default certificate assign to the application.
